examples/getting_started/3_Interactivity.ipynb

```diff
@@ -262,7 +262,7 @@
     "\n",
     "## HoloViews+Datashader+Panel\n",
     "\n",
-    "To interactively explore data in a dashboard, you can combine Panel with HoloViews and Datashader to create an interactive visualization that allows you to toggle aggregation methods, edit colormaps, and generally interact with the data through the use of widgets (50 lines of code):\n",
+    "To interactively explore data in a dashboard, you can combine Panel with HoloViews and Datashader to create an interactive visualization that allows you to toggle aggregation methods, edit colormaps, and generally interact with the data through the use of widgets (50 lines of code: https://examples.pyviz.org/datashader_dashboard/dashboard.html):\n",
```

Comment:
Ah, I see that the link is obvious in the source file, but not on the rendered page. In that case:

```suggestion
    "To interactively explore data in a dashboard, you can combine Panel with HoloViews and Datashader to create an interactive visualization that allows you to toggle aggregation methods, edit colormaps, and generally interact with the data through the use of widgets ([50 lines of code](https://examples.pyviz.org/datashader_dashboard/dashboard.html)):\n",
```